Return values from multiple secret fields

laszlojau opened the pull request:
- Add a new `secret_fields` variable and the ability to return values from multiple fields
- Add updated instructions to README.md
- Add instructions for using sensitive values in Packer

breed808 commented on README.md:
```diff
@@ -19,8 +19,8 @@ Then, run [`packer init`](https://www.packer.io/docs/commands/init).
     tss = {
-      version = ">= 0.1.0"
-      source  = "github.com/breed808/packer-plugin-tss"
+      version = ">= 0.2.0"
+      source  = "github.com/breed808/tss"
```
Could you rebase the commit to remove the version changes? This will need to be bumped in a separate commit & tag.

breed808 commented on version/version.go:
```diff
@@ -6,7 +6,7 @@ import (
 var (
 	// Version is the main version number that is being run at the moment.
-	Version = "0.1.0"
+	Version = "0.2.0"
```
Same as the previous comment: could you rebase the commit to remove the version changes? This will need to be bumped in a separate commit & tag.

breed808 reviewed (changes requested):
This is really good, thanks for submitting this! I'd prefer to leave the version bump to separate commit & tag, and have left a couple comments to that effect.

Once you've got those addressed I'll happily merge them and then bump the version number and add a new plugin release.

laszlojau commented:
Thank you. I removed the version bump as requested.
I have also removed the commit for hard-coding the version for `packer-plugins-check`. Apparently, that issue is fixed with [PR 11282](https://github.com/hashicorp/packer/pull/11282) and only affects go versions lower than 1.17. The build I just tried still failed though, so I believe the actual fix will come with Packer v1.7.6.
Anyway, I'd leave that decision up to you - as per my understanding, the current fix is either hard-coding `packer-plugins-check@v1.7.4` in `.goreleaser.yml` until v1.7.6 is released or bumping the go version to 1.17. For example, `packer-plugin-amazon` was recently bumped to go 1.17 with [this commit](https://github.com/hashicorp/packer-plugin-amazon/commit/424e8e8bc8c6e62b1a65f67fb812d64308c79fe8#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6).

nywilken commented:
> Thank you. I removed the version bump as requested. I have also removed the commit for hard-coding the version for `packer-plugins-check`. Apparently, that issue is fixed with [PR 11282](https://github.com/hashicorp/packer/pull/11282) and only affects go versions lower than 1.17. The build I just tried still failed though, so I believe the actual fix will come with Packer v1.7.6. Anyway, I'd leave that decision up to you - as per my understanding, the current fix is either hard-coding `packer-plugins-check@v1.7.4` in `.goreleaser.yml` until v1.7.6 is released or bumping the go version to 1.17. For example, `packer-plugin-amazon` was recently bumped to go 1.17 with [this commit](https://github.com/hashicorp/packer-plugin-amazon/commit/424e8e8bc8c6e62b1a65f67fb812d64308c79fe8#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6).

Hi 👋  apologies for the inconvenience. Packer v1.7.6 has been released. Please let us know if you are still running into issues here. Cheers!

laszlojau commented:
> > Thank you. I removed the version bump as requested. I have also removed the commit for hard-coding the version for `packer-plugins-check`. Apparently, that issue is fixed with [PR 11282](https://github.com/hashicorp/packer/pull/11282) and only affects go versions lower than 1.17. The build I just tried still failed though, so I believe the actual fix will come with Packer v1.7.6. Anyway, I'd leave that decision up to you - as per my understanding, the current fix is either hard-coding `packer-plugins-check@v1.7.4` in `.goreleaser.yml` until v1.7.6 is released or bumping the go version to 1.17. For example, `packer-plugin-amazon` was recently bumped to go 1.17 with [this commit](https://github.com/hashicorp/packer-plugin-amazon/commit/424e8e8bc8c6e62b1a65f67fb812d64308c79fe8#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6).
> 
> Hi 👋 apologies for the inconvenience. Packer v1.7.6 has been released. Please let us know if you are still running into issues here. Cheers!

Thank you, the build went through without issues this time.

breed808 reviewed (approved):
Thanks, this is great!
